docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,808 @@
+  | Commands:                                                                           |
+  +-------------------------------------------------------------------------------------+
+  | * command-by-id                                                                     |
+  | * read-by-id                                                                        |
+  | * read                                                                              |
+  | * write-by-id                                                                       |
+  | * write                                                                             |
+  | * subscribe-by-id                                                                   |
+  | * subscribe                                                                         |
+  | * read-event-by-id                                                                  |
+  | * subscribe-event-by-id                                                             |
+  +-------------------------------------------------------------------------------------+
+[1647502596.396299][411686:411686] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/Commands.cpp:84: Error 0x0000002F
+```
+
+According to this list, the `binding` cluster supports operations such as read
+or write. Attributes from that cluster can also be subscribed by the controller,
+which means that the CHIP Tool will receive notifications, for instance when the
+attribute value is changed or a specific event happens.
+
+#### Examples
+
+This section lists examples of the CHIP Tool commands dedicated to specific use
+cases.
+
+##### Writing ACL to the `accesscontrol` cluster
+
+The Access Control List (ACL) concept allows to govern all Data Model
+interactions (such as read attribute, write attribute, invoke command). For more
+information about ACL, see
+[Access Control Guide](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/access-control-guide.md).
+
+To write ACL to the `accesscontrol` cluster, use the following command pattern:
+
+```
+$ chip-tool accesscontrol write acl <acl_data> <node_id> <endpoint_id>
+```
+
+In this command:
+
+-   _<acl_data\>_ is the ACL data formatted as a JSON array.
+-   _<node_id\>_ is the ID of the node that is going to receive ACL.
+-   _<endpoint_id\>_ is the ID of the endpoint on which the `accesscontrol`
+    cluster is implemented.
+
+##### Adding a binding table to the `binding` cluster
+
+Binding describes a relationship between the device that contains the binding
+cluster and the end device. The proper ACL must be added to allow the end device
+to receive commands from the bonded device. After the binding process, the
+bonded device contains information about connected device, such as IPv6 address
+and the route to the endpoint in the Matter network. This allows to send command
+directly from the bonded device to the end device without using the border
```

Comment:
BR is not necessary. The communication happens just like in any other case, so it's not worth adding more constraints in this description.